vowpalwabbit/learner.h

```diff
@@ -32,13 +32,14 @@ enum class prediction_type_t
   multiclass,
   multilabels,
   prob,
-  multiclassprobs,
+  multiclassprobs,  // not in use?
```

Comment:
this seems to not be referenced anywhere

Reply:
Maybe thats the corresponding type for when multilabels is used as a prediction. Althought I cannot see it used anywhere

Reply:
```suggestion
  multiclassprobs,  // not in use (technically oaa.cc)
```